projects/cashmere/src/lib/modal/modal.component.ts

```diff
@@ -14,16 +13,6 @@ import { ModalHeaderComponent } from './modal-header.component';
 export class ModalComponent implements AfterContentInit {
     @HostBinding('class.hc-modal-content')
     _modalWrapperClass = true;
-    private _tight = false;
-
-    /** If true, condense the default padding on all elements and reduce the title font size. *Defaults to `false`.*  */
-    @Input()
-    get tight(): boolean {
-        return this._tight;
-    }
-    set tight(value: boolean) {
-        this._tight = parseBooleanAttribute(value);
-    }
```

Comment:
Turns out this input param actually does nothing 😬 looks like this was maybe added during development but never removed.  The only way to add tight to a modal is via `ModalOptions` like everything else.  So this is just confusing to developers to have a zombie input.

Reply:
Ooh good catch